sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -172,6 +166,12 @@ export class ConfirmationOfEnrollmentController {
       ),
       applicationLocationId: application.location.id,
       applicationDeniedReason: getCOEDeniedReason(application),
+      studyBreaks: application.offering.studyBreaks?.map((studyBreak) => {
```

Comment:
The return could be simplified as below.
```
application.offering.studyBreaks?.map((studyBreak) => ({
    breakStartDate: dateString(studyBreak.breakStartDate),
    breakEndDate: dateString(studyBreak.breakEndDate),
})),
```

Reply:
Code changed.